### ISSUE TITLE
Rate limit in Powershell

### ISSUE THREAD

**Issue by hygrace** [FirstStreet/fsf_api_access_python#30]
Hi there,

I tried the rate limit code but the powershell did not recognized it (please refer as below). Could you please let me know if there is anything I did incorrectly for queued request?
![error](https://user-images.githubusercontent.com/72878755/96011739-4df3fa80-0e11-11eb-9682-90422945397f.PNG)

Thank you very much in advance.

**Comment by Lyetenth** [comment]
Hi @hygrace,

Could you post your Python version and your `fsf-api-access-python` version? 

The version that includes the rate limit function is in `fsf-api-access-python version 2.1.0`

**Comment by hygrace** [comment]
![image](https://user-images.githubusercontent.com/72878755/96023596-abdc0e80-0e20-11eb-96fc-102e84cb137c.png)
Does this mean I have 2.2.1?

**Comment by Lyetenth** [comment]
Hi @hygrace,,

The `ratelimit` package looks good. Could you run `pip list` as well and post the output from that call?

**Comment by hygrace** [comment]
![image](https://user-images.githubusercontent.com/72878755/96025944-dbd8e100-0e23-11eb-8734-f553c6d90cf9.png)

![image](https://user-images.githubusercontent.com/72878755/96025879-c663b700-0e23-11eb-92b3-96cd44ed075a.png)
It has the same error

**Comment by Lyetenth** [comment]
Hi @hygrace,
<br>

So it looks like your `fsf-api-access-python` version is 2.0.1 from the top screenshot, the 6th package

Since it looks like you have version 2, you most likely also have Python 3.7+


To update to your `fsf-api-access-python` to 2.1.0, you can use a similar update command as in your second comment, then use the command again.
<br>
ie: 
```
cd /path/to/project

.\venv\Scripts\activate

pip install --upgrade fsf_api_access_python

python -m firststreet -p location.get_summary -f FSIDs_Tulsa.txt -l property -rate_limit 5000 rate_period 3600
```

**Comment by hygrace** [comment]
Thank you Lyetenth. It works now but within the call, sometimes the below error occurs.  Will it affect the final outcome?
![image](https://user-images.githubusercontent.com/72878755/96310926-55b0cc00-0fd6-11eb-9e54-d95c14e39145.png)

**Comment by Lyetenth** [comment]
Hi @hygrace,

I've been taking a look into the causes and I think it's because of the number of input FSIDs in combination of the loop. I created a fix (Version 2.1.1) that should solve the majority of the Timeout errors.

You MAY still see some Timeout Errors, but each error will be retried up to a max of 5 times before it skips that FSID. So if it doesn't say `Retry 5` at the end of the line, it should be successful still.

Update to the new update,

Ex:
```
cd /path/to/project

.\venv\Scripts\activate

pip install --upgrade fsf_api_access_python

python -m firststreet -p location.get_summary -f FSIDs_Tulsa.txt -l property -rate_limit 4950 rate_period 60 -connection_limit 100
```

 And let me know if there are any other issues

**Comment by Lyetenth** [comment]
Hey @hygrace ,

Closing this due to inactivity. Please feel free to reopen this if there are still issues with the ratelimit in the newest version
